config/settings.yml

```diff
@@ -4,6 +4,23 @@
     :blacklisted_event_names: []
     :event_handling:
       :event_groups:
+        :addition:
+          :critical_regex:
+          - ^nuage_.+_create$
+        :deletion:
+          :critical_regex:
+          - ^nuage_.+_delete$
+        :update:
+          :critical_regex:
+          - ^nuage_.+_update$
+          :detail:
```

Comment:
Please just remove `:detail:` category. We want all events to land in `:critical:` at this point and will decide which ones to degradate to `:detail:` later.

Reply:
Will do. I simply converted the code from the older pull request.